### DISCUSSION TITLE
Sub-collections

### DISCUSSION THREAD

**Discussion by MonicaGalassi** [MukurtuCMS/Mukurtu-CMS#28] [Mukurtu 4 Core Features for Launch]
I love (and our users as well) the sub-collections feature but I find the visualisation confusing as the "boxes" are too small. As a result, we avoid using it. I think a larger visualisation in the middle of the page, with larger boxes that are linked with the same "tree" system would be much more user-friendly. Happy to provide more ideas on this if that's something that is in the radar

**Comment by michael-wynne-wsu** [comment]
I think collections as a whole are due for a visual overhaul. 
I could use more detail/ a sketch of what you have in mind, I don't think I'm picturing it quite right...
